src/couch/test/eunit/couch_doc_tests.erl

```diff
@@ -115,8 +115,8 @@ mock_config() ->
         config,
         get,
         fun
-            ("couchdb", "max_document_id_length", "infinity") -> "1024";
-            ("couchdb", "max_attachment_size", "infinity") -> "infinity";
+            ("couchdb", "max_document_id_length", _) -> 238;
```

Comment:
This should be `"238"`. Because `config:get/3` return lists.

Reply:
Thank you for the fix! See https://github.com/apache/couchdb/pull/3632/commits/468123da973d22ff9d4e91477b1f48568cde80b9.